spec/controllers/faye_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe FayeController do
+  include Devise::TestHelpers
+  before :each do
+    user = create :user, admin: true
```

Comment:
See, this is why I wrote a helper method for doing this in one line. Wouldn't it be nicer to just write

```ruby
when_current_user_is :admin
```

Could we update that spec_helper method to use the devise test helper?